index.html

```diff
@@ -43,7 +43,7 @@ <h3>LET'S CONNECT</h3>
                                     src="/images/github.png" alt="github
                                     link"></a></li>
                         <li class="icon"><a href="#"><img
-                                    src="/images/doits.png" alt="doit"></a></li>
+                                    src="/images/doits.png" alt="doits link"></a></li>
```

Comment:
[OPTIONAL] I suggest it's maybe better to use descriptive alt tags on images because this helps people with sight imperatives, they might not understand what is 'doits' as i also don't 👍